wolfhsm/wh_packet.h

```diff
@@ -29,11 +29,22 @@
     #define WOLFHSM_PACK
 #endif
 
+#ifdef WOLFHSM_PACK
+#undef WOLFHSM_PACK
+#define WOLFHSM_PACK
+#endif
```

Comment:
Looks like you went ahead and padded all the structs in this file, thus eliminating the need for `WOLFHSM_PACK`. Can you just remove all traces of this macro entirely then?